playbooks/templates/k8s/jail/matomo-jail.yml.j2

```diff
@@ -167,6 +167,20 @@ data:
         ## only allow accessing the following php files
         location ~ ^/(index|plugins/HeatmapSessionRecording/configs)\.php {
 {% endif %}
+
+{% if matomo_name == 'matomo-api' %}
+          access_by_lua_block {
+            local opts = {
+              defaultCsp = "https://{{ wai_hostname[matomo_ingress_fqdn_index] }}.{{ wai_base_domain[item] }}",
```

Comment:
`{{ wai_hostname[matomo_ingress_fqdn_index] }}.` non dovrebbe essere nel CSP di default.

Reply:
Corretto questo è il CSP di default che uso come configurazione da portare dentro allo script LUA.

Reply:
@giafar ma la variabile `defaultCsp` non dovrebbe essere uguale al fqdn del portale WAI nello specifico env?
Quindi qualcosa come `"https://{{ wai_base_domain[item] }}"`? Era questo che intendevo nel primo commento.

Reply:
In realtà dovrebbe essere:

1. Se mi trovo su https://webanalytics.italia.it --> https://webanalytics.italia.it https://api.webanalytics.italia.it
1. Se mi trovo su https://www.pa.it --> https://pa.it https://api.webanalytics.italia.it

Modificato quindi per inserire di default sempre https://webanalytics.italia.it https://api.webanalytics.italia.it